### PR TITLE
Save signup fields

### PR DESCRIPTION
**Pull request by jamakase**
## What
Stores signup Fields in firebase entity and uses them later when creating the user.

### CODE REVIEW

File: airbyte-webapp/src/packages/cloud/lib/auth/GoogleAuthService.ts [hunk 1]
```diff
@@ -89,6 +90,13 @@ export class GoogleAuthService implements AuthService {
     );
   }
 
+  async updateProfile(displayName: string): Promise<void> {
```
Comment (timroes) [inline airbyte-webapp/src/packages/cloud/lib/auth/GoogleAuthService.ts:93]: Do we still need this method, or can we remove it if unused?

File: airbyte-webapp/src/packages/cloud/services/auth/AuthService.tsx [hunk 2]
```diff
@@ -64,6 +68,45 @@ type AuthContextApi = {
...
+      name: currentUser.email ?? "",
+      news: false,
+    };
+  },
+  save: async (
+    currentUser: FbUser,
+    v: { companyName: string; name: string; news: boolean }
+  ) => {
+    localStorage.setItem(
+      `${currentUser.uid}/temp-signup-data`,
```
Comment (timroes) [inline airbyte-webapp/src/packages/cloud/services/auth/AuthService.tsx:104]: :art: We could also use `getTempSignUpStorageKey` here.

File: airbyte-webapp/src/packages/cloud/services/auth/AuthService.tsx [hunk 3]
```diff
@@ -64,6 +68,45 @@ type AuthContextApi = {
 
 export const AuthContext = React.createContext<AuthContextApi | null>(null);
 
+const getTempSignUpStorageKey = (currentUser: FbUser): string =>
+  `${currentUser.uid}/temp-signup-data`;
+const TempSignUpValuesProvider = {
+  get: async (
```
Comment (timroes) [inline airbyte-webapp/src/packages/cloud/services/auth/AuthService.tsx:74]: :art: Wouldn't need to be `async` anymore. Same for the `save` method.

**Review by timroes — APPROVED** [review]
Code LGTM. Haven't tested locally.

File: airbyte-webapp/src/packages/cloud/lib/auth/GoogleAuthService.ts (hunk 1 above)
Comment (Jamakase) [reply airbyte-webapp/src/packages/cloud/lib/auth/GoogleAuthService.ts:93]: It's not used anymore, but let's keep it as it was already added.

File: airbyte-webapp/src/packages/cloud/services/auth/AuthService.tsx (hunk 2 above)
Comment (Jamakase) [reply airbyte-webapp/src/packages/cloud/services/auth/AuthService.tsx:104]: Oh, exactly! - that's why that method was added.

File: airbyte-webapp/src/packages/cloud/services/auth/AuthService.tsx (hunk 3 above)
Comment (Jamakase) [reply airbyte-webapp/src/packages/cloud/services/auth/AuthService.tsx]: Yeah, thought that there is no need to change the interface, but actually, as this method is only for temporary usage - let's get rid of `async` to make it clearer.

**Pull request merged**